packages/core/src/domain/configuration/experimentalFeatures.ts

```diff
@@ -0,0 +1,9 @@
+let enabledExperimentalFeatures: Set<string>
+
+export function setEnabledExperimentalFeatures(enabledFeatures: string[]): void {
+  enabledExperimentalFeatures = new Set(enabledFeatures)
+}
+
+export function isEnabledExperimentalFeature(featureName: string): boolean {
+  return enabledExperimentalFeatures.has(featureName)
+}
```

Comment:
I have some thoughts about the naming. What do you think about:

- enableExperimentalFeatures(enabledFeatures)
- isExperimentalFeatureEnabled(featureName)

Reply:
This convention of {specifier + group_name} is less correct english, but it isn't uncommon on codebases.
I don't mind one way or the other. Do we have a convention of the reverse?